src/aapns/api.py

```diff
@@ -102,6 +132,19 @@ class APNS:
     *,
     logger: Optional[BoundLogger] = None,
     timeout: TimeoutTypes = DEFAULT_TIMEOUT_CONFIG,
+    cafile: str = None,
 ) -> APNS:
-    client = AsyncClient(http2=True, cert=client_cert_path, timeout=timeout)
-    return APNS(client, logger, server)
+    client = AsyncClient(
+        http2=True,
+        cert=client_cert_path,
+        timeout=timeout,
+        **{"verify": cafile} if cafile else {},
+    )
+    base_url = f"https://{server.host}:{server.port}"
+    ssl_context = create_ssl_context()
+    if cafile:
+        ssl_context.load_verify_locations(cafile=cafile)
+    ssl_context.load_cert_chain(certfile=client_cert_path, keyfile=client_cert_path)
+    apns = APNS(client, logger, server, Pool(base_url, ssl=ssl_context))
+    await apns.pool.__aenter__()
```

Comment:
🤔 this feels kinda hacky to me, is it possible to have an alias function that basically calls this?

Reply:
I think the client should get an already valid/entered pool.

Reply:
All re-done, closing the comment.